app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -141,10 +142,18 @@
     private SessionNavigator sessionNavigator;
     private FormAndDataSyncer formAndDataSyncer;
 
+    private boolean loginExtraWasConsumed;
+    private static final String EXTRA_CONSUMED_KEY = "login_extra_was_consumed";
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        if (savedInstanceState != null) {
+            wasExternal = savedInstanceState.getBoolean(WAS_EXTERNAL_KEY);
```

Comment:
Ah, I wasn't paying close enough attention when I refactored this; I think it makes more sense to leave it in processFromExternalLaunch so I'll remove it from here.